book/installation.rst

```diff
@@ -98,6 +98,17 @@ number as the second argument of the ``new`` command:
     # Windows
     c:\projects\> php symfony.phar new my_project_name 2.3.23
 
+If you want your project to be based on the last Symfony LTS version, pass
```

Comment:
Latest could be interpreted as "they are a lot of LTS versions", but there is only one
For me last is ok, or nothing at all